site/tests/conftest.py

```diff
@@ -8,17 +8,40 @@
 """Pytest fixtures."""
 
 from collections import namedtuple
+from copy import deepcopy
```

Comment:
It feels like we need RDM fixtures in pytest-invenio (or somewhere else), to avoid copy/pasting all this around.